index.js

```diff
@@ -13,12 +13,14 @@ const MAX_TAG_VALUE_LENGTH = 256;
 const SANITIZATION_CHARACTER = '_';
 const ROLE_SESSION_NAME = 'GitHubActions';
 const REGION_REGEX = /^[a-z0-9-]+$/g;
+const DEFAULT_AUDIENCE = 'sts.amazonaws.com';
```

Comment:
if you move this to the default in action.yml, it simplifies the logic below (just always pull the value form the action config)